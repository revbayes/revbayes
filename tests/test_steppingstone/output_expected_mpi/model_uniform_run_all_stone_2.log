Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21332.58	-21301.04	-31.53277	0.7237501	0.146326	0.125838	0.187767	0.149528	0.235907	0.263473	0.460427	0.168528	0.178706	0.193476	0.309898	0.230861	0.144514	0.214834	0.208056	0.0991044	0.154861	0.214217	0.208794	0.273659	0.360108	0.182258	0.773275	0.201197	0.0517493	0.00326358	0.133294	0.068935	0.0649893	0.180021	0.278817	0.0823588	0.140896	0.00326539	0.0958125	0.0933458	0.0696482	0.0528208	0.027077	0.0099199	0.220591	0.186163	0.324099	0.0267166	0.398497	0.0518595	0.0304598	0.428578	0.0638899	0.0792668	0.374524	0.933361	2.61285	0.374226	0.330691	0.0828142	0.212268	0.3693284	7.772677
10	-21316.74	-21292.28	-24.46165	0.6145064	0.175621	0.126362	0.223853	0.144501	0.176875	0.249746	0.284722	0.0959109	0.216244	0.10201	0.244716	0.199591	0.109039	0.214753	0.176462	0.140711	0.125967	0.193768	0.214882	0.277554	0.437165	0.231218	0.663963	0.106543	0.0675722	0.00107059	0.111387	0.0857425	0.0419358	0.173344	0.333671	0.0651804	0.0798228	0.0063207	0.0600886	0.0304485	0.0602786	0.112237	0.0389878	0.016054	0.226193	0.163506	0.223828	0.0298024	0.392783	0.0399466	0.0254421	0.455411	0.0566153	0.0561131	0.32024	0.888543	2.7351	0.365765	0.326323	0.0733437	0.234568	0.3690086	7.029845
