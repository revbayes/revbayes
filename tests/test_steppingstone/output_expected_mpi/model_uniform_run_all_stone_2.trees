Iteration	Posterior	Likelihood	Prior	psi
0	-21332.58	-21301.04	-31.53277	(Galeopterus_variegatus[&index=23]:0.773275,(((((Lemur_catta[&index=10]:0.193476,Varecia_variegata_variegata[&index=22]:0.182258)[&index=38]:0.052821,Propithecus_coquereli[&index=19]:0.208794)[&index=40]:0.009920,(Lepilemur_hubbardorum[&index=11]:0.309898,(Cheirogaleus_major[&index=4]:0.149528,Microcebus_murinus[&index=14]:0.214834)[&index=35]:0.095812)[&index=39]:0.027077)[&index=27]:0.133294,((Nycticebus_coucang[&index=15]:0.208056,Loris_tardigradus[&index=12]:0.230861)[&index=37]:0.069648,(Perodicticus_potto[&index=18]:0.214217,(Otolemur_crassicaudatus[&index=16]:0.099104,Galago_senegalensis[&index=8]:0.168528)[&index=24]:0.201197)[&index=29]:0.064989)[&index=41]:0.220591)[&index=28]:0.068935,(Tarsius_syrichta[&index=21]:0.360108,(((Pan_paniscus[&index=17]:0.154861,Hylobates_lar[&index=9]:0.178706)[&index=32]:0.082359,(Colobus_guereza[&index=6]:0.263473,(Chlorocebus_aethiops[&index=5]:0.235907,Macaca_mulatta[&index=13]:0.144514)[&index=25]:0.051749)[&index=42]:0.186163)[&index=30]:0.180021,(((Cebus_albifrons[&index=3]:0.187767,Aotus_trivirgatus[&index=1]:0.146326)[&index=26]:0.003264,Saimiri_sciureus[&index=20]:0.273659)[&index=33]:0.140896,Callicebus_donacophilus[&index=2]:0.125838)[&index=43]:0.324099)[&index=31]:0.278817)[&index=36]:0.093346)[&index=34]:0.003265,Daubentonia_madagascariensis[&index=7]:0.460427)[&index=44];
10	-21316.74	-21292.28	-24.46165	(Galeopterus_variegatus[&index=23]:0.663963,Daubentonia_madagascariensis[&index=7]:0.284722,(((((Otolemur_crassicaudatus[&index=16]:0.140711,Galago_senegalensis[&index=8]:0.095911)[&index=24]:0.106543,Perodicticus_potto[&index=18]:0.193768)[&index=29]:0.041936,(Nycticebus_coucang[&index=15]:0.176462,Loris_tardigradus[&index=12]:0.199591)[&index=37]:0.060279)[&index=41]:0.226193,(((Lemur_catta[&index=10]:0.102010,Varecia_variegata_variegata[&index=22]:0.231218)[&index=38]:0.112237,Propithecus_coquereli[&index=19]:0.214882)[&index=40]:0.016054,(Lepilemur_hubbardorum[&index=11]:0.244716,(Microcebus_murinus[&index=14]:0.214753,Cheirogaleus_major[&index=4]:0.144501)[&index=35]:0.060089)[&index=39]:0.038988)[&index=27]:0.111387)[&index=28]:0.085743,(Tarsius_syrichta[&index=21]:0.437165,(((Colobus_guereza[&index=6]:0.249746,(Macaca_mulatta[&index=13]:0.109039,Chlorocebus_aethiops[&index=5]:0.176875)[&index=25]:0.067572)[&index=42]:0.163506,(Pan_paniscus[&index=17]:0.125967,Hylobates_lar[&index=9]:0.216244)[&index=32]:0.065180)[&index=30]:0.173344,(((Cebus_albifrons[&index=3]:0.223853,Aotus_trivirgatus[&index=1]:0.175621)[&index=26]:0.001071,Saimiri_sciureus[&index=20]:0.277554)[&index=33]:0.079823,Callicebus_donacophilus[&index=2]:0.126362)[&index=43]:0.223828)[&index=31]:0.333671)[&index=36]:0.030449)[&index=34]:0.006321)[&index=44];
