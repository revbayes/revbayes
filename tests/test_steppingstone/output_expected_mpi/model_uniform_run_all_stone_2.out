state	power	likelihood
4	0.32768	-21310.7
5	0.32768	-21310.1
6	0.32768	-21293
7	0.32768	-21295.8
8	0.32768	-21292.7
9	0.32768	-21292.2
10	0.32768	-21292.3
